apogee/tools/read.py

```diff
@@ -132,19 +135,19 @@ def allStar(rmcommissioning=True,
         download.allStar()
     #read allStar file
     #data= pyfits.getdata(path.allStarPath())
-    data= fitsio.read(path.allStarPath())
+    data= fitsread(path.allStarPath())
     if raw: return data
     #Remove duplicates, cache
     if rmdups:
         dupsFilename= path.allStarPath().replace('.fits','-nodups.fits')
         if os.path.exists(dupsFilename):
-            data= fitsio.read(dupsFilename)
+            data= fitsread(dupsFilename)
         else:
             sys.stdout.write('\r'+"Removing duplicates (might take a while) and caching the duplicate-free file ...\r")
             sys.stdout.flush()
             data= remove_duplicates(data)
             #Cache this file for subsequent use of rmdups
-            fitsio.write(dupsFilename,data,clobber=True)
+            fitswrite(dupsFilename,data,clobber=True)
```

Comment:
Thanks! Does this line work in pyfits?

Reply:
I haven't explicitly tested it, since remove_duplicates needs esutils and I can't figure out how to get that working locally. However, pyfits.writeto has the same format as fitsio.write e.g. pyfits.writeto(filename, data, header=None, output_verify='exception', clobber=False, checksum=False), so it seems like it should work.

Reply:
Okay, thanks. Wasn't sure about the ``clobber`` keyword...